assets/src/date/getRelativeDisplayDate.js

```diff
@@ -64,5 +64,5 @@ export function getRelativeDisplayDate(
 
   const fmt = new DateFormatter();
 
-  return fmt.formatDate(date.toDate(), dateSettings.dateFormat);
+  return fmt.formatDate(displayDate.toDate(), dateSettings.dateFormat);
```

Comment:
@BrittanyIRL I believe you're our date expert, does this change make sense?  One of our karma tests were breaking because `date` was being passed in as pure string and `toDate()` doesn't exist on a string object, but I believe `displayDate` is the object that we want to use (since it's the same date just converted to a moment object).  I also added an extra test to the `getRelativeDisplayDate.js` tests to test this scenario.

Reply:
I think it does. I‘ve fixed it in another date-related PR that‘s still WIP but I can hopefully merge soon.